R/pkg/R/SQLContext.R

```diff
@@ -254,6 +301,7 @@ jsonFile <- function(sqlContext, path) {
 #' df <- jsonRDD(sqlContext, rdd)
 #'}
 
+# TODO: remove - this method is no longer exported
```

Comment:
Can we open a JIRA for this ? Would be good to clean up this file as I think a bunch of functions are not exported here.